samples/file-encryptor/Makefile

```diff
@@ -3,6 +3,7 @@
 
 .PHONY: all build clean run simulate
 
+# Set OE_CRYPTO_LIB as "mbedtls" or "openssl", to use the respective cryptolibs
```

Comment:
Line 36 of samples/README.md explicitly says that the file-encryptor sample does not support openssl yet. If this PR is adding that support, then this PR should update that README file too

Reply:
removed comment.